src/model/car.js

```diff
@@ -0,0 +1,23 @@
+import { CAR } from '../utils/constants.js';
+
+export class Car {
+  #name;
+  #currentLocation;
+
+  constructor(name) {
+    this.#name = name;
+    this.#currentLocation = CAR.DEFAULT_LOCATION;
+  }
+
+  move() {
+    this.#currentLocation++;
+  }
+
+  getName() {
+    return this.#name;
+  }
```

Comment:
사소한 부분이지만 접근자 프로퍼티를 사용하면 car.name 이렇게 public 속성에 접근하는 것처럼 getter를 만들 수 있습니다!

https://developer.mozilla.org/ko/docs/Web/JavaScript/Reference/Functions/get

```suggestion
  get name() {
    return this.#name;
  }
  
  // car.name 이렇게 접근
```